packages/vpnkit/vpnkit.0.0.0/opam

```diff
@@ -0,0 +1,61 @@
+opam-version: "1.2"
+maintainer: "David Scott <dave.scott@docker.com>"
+authors: [
+	"Anil Madhavapeddy <anil@recoil.org>"
+	"David Scott <dave.scott@docker.com>"
+	"David Sheets <dsheets@docker.com>"
+	"Gaetan de Villele <gdevillele@gmail.com>"
+	"Ian Campbell <ian.campbell@docker.com>"
+	"Magnus Skjegstad <magnus@skjegstad.com>"
+	"Mindy Preston <mindy.preston@docker.com>"
+	"Sebastiaan van Stijn <github@gone.nl>"
+	"Thomas Gazagnaire <thomas@gazagnaire.com>"
+	"Thomas Leonard <thomas.leonard@docker.com>"
+]
+homepage:     "https://github.com/docker/vpnkit"
+bug-reports:  "https://github.com/docker/vpnkit/issues"
+dev-repo:     "https://github.com/docker/vpnkit.git"
+doc:          "https://docker.github.io/vpnkit/"
+
+build: [
+  [make]
+]
+build-test: [
+  ["oasis" "setup"]
+  ["./configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-test"]
+]
+install: [make "install" "BINDIR=%{bin}%"]
+remove: [make "uninstall" "BINDIR=%{bin}%"]
+
+depends: [
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "oasis" {build}
+  "result"
+  "tar-format"
+  "ipaddr"
+  "lwt"
+  "uwt" { = "0.0.3" }
+  "tcpip" { >= "2.8.0" & < "3.0.0" }
+  "pcap-format"
+  "dns"
+  "dns-forward"
+  "datakit-server"
+  "hashcons" {= "1.0.1"}
+  "pcap-format" { >= "0.4.0" }
+  "cmdliner"
+  "charrua-core" { >= "0.3" }
+  "named-pipe" { >= "0.4.0" }
+  "hvsock"
+  "asl"
+  "win-eventlog"
+  "fd-send-recv"
+  "logs"
+  "fmt"
+  "astring"
+  "mirage-flow" { >= "1.1.0" }
+  "mirage-types-lwt"
+  "ounit"
+  "alcotest"
```

Comment:
this should probably be a `{test}` dependency (and same for `ounit`). Also not sure why `ounit` is still there, are still using it?